### PR TITLE
Main page header

### PR DESCRIPTION
**Pull request by artzhylindev**
- Added a mobile version of the SubHeader component

- Added a mobile version of the Popular component

- A common refactor of components and styles is done

### CODE REVIEW

File: frontend/components/BurgerMenu/burgerMenu.module.css [hunk 1]
```diff
@@ -3,6 +3,8 @@
 }
 
 .backgraund_burger {
```
Comment (Nordok) [inline frontend/components/BurgerMenu/burgerMenu.module.css:5]:
```suggestion
.background_burger {
```

File: frontend/hof/withRedux.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 import {initializeStore, removeUndefined} from "../store";
+import {categoriesApi} from "../store/categories/action";
 
 export const withRedux = getServerSideProps => async (ctx) => {
     const store = initializeStore();
     const {dispatch} = store;
     const result = await getServerSideProps(ctx, dispatch)
+    await dispatch(categoriesApi.endpoints.getAllCategories.initiate())
```
Comment (Nordok) [inline frontend/hof/withRedux.js:8]: Why did you add it here?

File: frontend/components/BurgerMenu/burgerMenu.module.css (hunk 1 above)
Comment (artzhylindev) [reply frontend/components/BurgerMenu/burgerMenu.module.css]: Fixed in commit Fix backgraund -> background
